backend/core/src/listings/combined/combined-listing.entity.ts

```diff
@@ -0,0 +1,392 @@
+import { UnitsSummarized } from "../../units/types/units-summarized"
+import { PrimaryColumn, ViewColumn, ViewEntity } from "typeorm"
+import { ListingFeatures } from "../entities/listing-features.entity"
+import { ListingUtilities } from "../entities/listing-utilities.entity"
+import { Address } from "../../shared/entities/address.entity"
+import { Jurisdiction } from "../../jurisdictions/entities/jurisdiction.entity"
+import { ReservedCommunityType } from "../../reserved-community-type/entities/reserved-community-type.entity"
+import { ListingMultiselectQuestion } from "../../multiselect-question/entities/listing-multiselect-question.entity"
+import { ListingImage } from "../entities/listing-image.entity"
+import { User } from "../../auth/entities/user.entity"
+import { Unit } from "../../units/entities/unit.entity"
+import { ListingReviewOrder, ListingStatus } from "../../../types"
+import { AssetCreateDto } from "../../assets/dto/asset.dto"
+
+const localSelect = `SELECT
```

Comment:
I think this is ok for this use case. It's probably more performant than using an SQL generator. Like you pointed out the biggest issue with going this route is that any downstream changes to the model would break this so we have to be careful on what is changed. However, since it's a short term solution I think this is better than trying to over-engineer it at this point.

@YazeedLoonat do you have any issues with this view query?

Reply:
Per convo with Morgan, current state is fine for now.  We can circle back on this in the future if needed, but given the short lifetime of this code it may not be needed